selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindTextFeatureTest.java

```diff
@@ -442,51 +440,51 @@ public void checkTextResultsPagination() {
 
     // Check move page buttons status on the first page
     findText.sendCommandByKeyboardInFindInfoPanel(ARROW_DOWN.toString());
-    assertTrue(findText.checkNextPageButtonIsEnabled());
-    assertFalse(findText.checkPreviousPageButtonIsEnabled());
-    sumOfFoundFiles += findText.getFoundFilesNumberOnPage();
-    sumOfFoundOccurrences += findText.getFoundOccurrencesNumberOnPage();
+    Assert.assertTrue(findText.checkNextPageButtonIsEnabled());
+    Assert.assertFalse(findText.checkPreviousPageButtonIsEnabled());
+    sumOfFoundFiles += findText.getResults().getFoundFilesOnPage();
```

Comment:
It's better to set the returning by `findText.getResults()` object to the variable and then use it to get the numbers.